addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -254,22 +250,37 @@ registerModel({
          * @private
          */
         _prepareAttachmentsLoading() {
-            this._isPreparingAttachmentsLoading = true;
-            this._attachmentsLoaderTimeout = this.env.browser.setTimeout(() => {
-                this.update({ isShowingAttachmentsLoading: true });
-                this._isPreparingAttachmentsLoading = false;
-            }, this.messaging.loadingBaseDelayDuration);
+            this.update({
+                _attachmentsLoaderTimeout: this.env.browser.setTimeout(
+                    () => {
+                        this.update({
+                            isShowingAttachmentsLoading: true,
+                            _isPreparingAttachmentsLoading: clear(),
+                        });
+                    },
+                    this.messaging.loadingBaseDelayDuration,
+                ),
+                _isPreparingAttachmentsLoading: true,
+            });
         },
         /**
          * @private
          */
         _stopAttachmentsLoading() {
             this.env.browser.clearTimeout(this._attachmentsLoaderTimeout);
-            this._attachmentsLoaderTimeout = null;
-            this._isPreparingAttachmentsLoading = false;
+            this.update({
+                _attachmentsLoaderTimeout: clear(),
+                _isPreparingAttachmentsLoading: clear(),
+            });
         },
     },
     fields: {
+        _attachmentsLoaderTimeout: attr({
```

Comment:
Just kept the exact same name so the change is easier to digest. But I have no desire to introduce private/public fields. Field names are just names.

Reply:
Then I would say we should rename them immediately because this is confusing.

Also this will highlight them on the diff, which will help make sure we didn't forget to adapt update() for any of them.